conformance/utils/tester/tester.go

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
What do you think about adding tests about what this new interface/implementation has been created for? Something like setting logging headers and checking they are actually used. I think this would be beneficial for those who may want to use this functionality as well as a reference.

Reply:
Hi @mlavacca, adding tests for an interface feels a bit odd since it's a well-defined language feature.  Are you suggesting that I should implement some sort of default logging header inside `tester.impl`?

Reply:
Sorry for the delay, looks good to me!